src/components/BusinessViews/BusinessView.tsx

```diff
@@ -0,0 +1,46 @@
+import DescriptionView from "@components/BusinessViews/DescriptionView";
+import ImagesCarousel from "@components/BusinessViews/ImagesCarousel";
+import RatingsView from "@components/BusinessViews/RatingsView";
+import TitleView from "@components/BusinessViews/TitleView";
+import { BusinessType, DisplayableBusiness } from "@futureproof/typings";
+import React from "react";
+import { SafeAreaView, ScrollView, StyleSheet } from "react-native";
+
+// PLACEHOLDER BUSINESS -> Update to fetch graphQL
```

Comment:
Now that we have a solution to this problem, it would be a great time to get this working.